### ISSUE TITLE
Player and all mobs going invisible

### ISSUE THREAD

**Issue by C0D3-M4513R** [Asek3/Rubidium#276]
The player and all mobs sometimes go invisible.
I am playing ATM7 v.0.4.30 with Not Enough Crashes and Rubidium 0.5.3a
![grafik](https://user-images.githubusercontent.com/28912031/190859018-7835ab78-ee0e-4379-95cd-545abad33883.png)

**Comment by quentin452** [comment]
I tried atm 7 V0.4.31 with not enough crashes and rubidium and i can watch mobs and players
![Capture d’écran_2022-09-17_15-42-47](https://user-images.githubusercontent.com/42176772/190860046-da9525e5-f970-4e28-9544-6865db574093.png)

**Comment by quentin452** [comment]
How much ram allocated?

**Comment by C0D3-M4513R** [comment]
I usually allocate 10Gb. Also see the sometimes in the description. It happens after a longer time playing and having the game open. (I never really close the game, because I don't wanna wait for it to start again)

EDIT: The max in the launcher is actually configured at 16G

**Comment by wisheth** [comment]
Getting this as well with 5GB allocated. For me, this seems to happen on and off occasionally (across multiple clients with the same modpack installed) and is solved by a restart; is this similar?

**Comment by C0D3-M4513R** [comment]
Yes, I'm seeing something roughly similar. I've not tried multiple clients though.
Also I noticed that items in the Tinkers table for making ingots, and when pouring out fluids out of a smeltery, they are not visible.

**Comment by coderepo8593** [comment]
This probably isn't it but is worth a shot. If you turn off entity culling does it persist?

Also, what do you mean by "sometimes go invisible"? Is it after a certain period of time? Is it right when a new chunk is loaded? Does it ever happen when a chunk has been loaded for a while? Does it only happen with long play sessions? Please provide more information in this vein.

**Comment by C0D3-M4513R** [comment]
It is usually after a certain period of time. I can't tell you how long though. Try starting a game, and leaving it for 8-10h. It doesn't seem to matter if I stay within a couple chunks, or If I actively explore. That picture was me going through the nether, but I also had that happen to me in my base (I didn't even notice at first. Taking randomly damage gave it away eventually though).

Also I mentioned [here](https://github.com/Asek3/Rubidium/issues/276#issuecomment-1250102824), that it only happens after longer sessions.

**Comment by Shibva** [comment]
I have encountered this issue and it seems to be more than that actually; for me not only did that happen but also seem to have caused some rendering the seats with some of the tile entities; no exactly sure which ones were affected but I can tell you that the ones that were added by quark that were chests ceased running. And that actually happened to me very recently again when I joined a world

If I have to take a guess, whatever thread is running the entity rendering ran into some thing and caused it to cease in a non-fatal error

if this is true then it seems to happen whenever there's a re-render running or a chance of happening (events were the entire world render is reloaded as if it was you adjusting your POV settings or setting it between fast and fancy)

If that part of the code was well-built then there's probably some thing that's causing it to come to a hard stop


Only way to see if you can try and fix this bug with evidence is the hope that it happens to you again and once it does immediately pause your game and look into the game log and pray that it a rendering error was posted into it
